source/blog/2018-09-28-the-ember-times-issue-66.md

```diff
@@ -52,6 +52,18 @@ There has been [some documentation missing](https://github.com/emberjs/ember.js/
 
 Going forward the plan is to provide better testing for cases like this so you always have access to the documentation you need.
 
+---
+
+## Project Structure for Single Page Apps
+
+Inspired by Module Unification, [@NullVoxPopuli](https://github.com/NullVoxPopuli) has
```

Comment:
In our blog post we like to highlight some important phrases or keywords by **bold** font-styles per paragraph. Do you think there are a couple of keywords that could be highlighted in this section as well? E.g. 

```
**Module Unification**
```

or 

```
[...] article on **general project structure for Single Page Apps.**
```

What do you think?

Reply:
works for me!